app/src/main/scala/org/openzipkin/dependencies/spark/cassandra/ZipkinDependenciesJob.scala

```diff
@@ -61,13 +61,18 @@ object ZipkinDependenciesJob {
 
   val keyspace = sys.env.getOrElse("ZIPKIN_KEYSPACE", "zipkin")
   val cassandraHost = sys.env.getOrElse("CASSANDRA_HOST", "127.0.0.1")
+  val cassandraUser = sys.env.getOrElse("CASSANDRA_USER", "")
```

Comment:
mind changing these to `CASSANDRA_USERNAME` and `CASSANDRA_PASSWORD`? We'll eventually turn this into docker, and env variables should match existing ones.

https://github.com/openzipkin/zipkin/blob/master/zipkin-cassandra/README.md
https://github.com/openzipkin/zipkin/blob/master/zipkin-query-service/config/query-cassandra.scala#L30